scipy/sparse/linalg/interface.py

```diff
@@ -338,6 +340,56 @@ def matmat(self, X):
 
         return Y
 
+    def rmatmat(self, X):
+        """Adjoint matrix-matrix multiplication.
+
+        Performs the operation y = A^H * x where A is an MxN linear
+        operator and x is a column vector or 1-d array, or 2-d array.
+        The default implementation defers to the adjoint.
+
+        Parameters
+        ----------
+        x : {matrix, ndarray}
```

Comment:
Should probably be `X` instead of `x` to match the signature; the rest of the docstring should also be updated.

Reply:
Sorry, I forgot to update the docs. Do it now